Makefile

```diff
@@ -192,6 +192,13 @@ compile-documentation-scripts:
 compile-samples:
 	${ROOTDIR}/node_modules/.bin/tsc --project ./documentation/samples/tsconfig.json
 
+.PHONY: validate-samples
+validate-samples:
+	find documentation/samples/packs -name "*.ts" | \
+	parallel -k \
+	'echo "Validating {}"; node -r ts-node/register --no-deprecation cli/coda.ts validate {}'
```

Comment:
Is it `--no-checkDeprecationWarnings`?

Reply:
I can't find any reference to that. But the docs do include `--no-deprecation`: https://nodejs.org/api/cli.html#--no-deprecation

Reply:
Ah I see, we're doing different things here. `checkDeprecationWarnings` is a Coda-specific flag for checking SDK deprecation warnings. I guess it's good to keep checking those in sample code. (In the `packs` repo, we have deprecated syntax everywhere so it's not meaningful to check it today.) You're ignoring deprecated stuff at the `node` level, so yeah keep doing what you're doing.